docsrc/getting_started/jetpack.rst

```diff
@@ -0,0 +1,117 @@
+.. _Torch_TensorRT_in_JetPack_6.1
+
+Overview
+##################
+
+JetPack 6.1
+---------------------
+Nvida JetPack 6.1 is the latest production release ofJetPack 6.
+With this release it incorporates:
+CUDA 12.6
+TensorRT 10.3
+cuDNN 9.3
+DLFW 24.0
+
+You can find more details for the JetPack 6.1:
+
+    * https://docs.nvidia.com/jetson/jetpack/release-notes/index.html
+    * https://docs.nvidia.com/deeplearning/frameworks/install-pytorch-jetson-platform/index.html
+
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+
+Ensure your jetson devleoper kit has been flashed with the latest JetPack 6.1. You can find more details on how to flash Jetson board via sdk-manager:
+
+    * https://developer.nvidia.com/sdk-manager
+
+
+check the current jetpack version using
+
+.. code-block:: sh
+
+    apt show nvidia-jetpack
+
+Ensure you have installed JetPack Dev components. This step is required if you need to build on jetson board.
+
+You can only install the dev components that you require: ex, tensorrt-dev would be the meta-package for all TRT development or install everthing.
+
+.. code-block:: sh
+    # install all the nvidia-jetpack dev components
+    sudo apt update
+    sudo apt install nvidia-jetpack
+
+Ensure you have cuda 12.6 installed(this should be installed automatically from nvidia-jetpack)
+
+.. code-block:: sh
+
+    # check the cuda version
+    nvcc --version
+    # if not installed or the version is not 12.6, install via the below cmd:
+    sudo apt update
+    sudo apt install cuda-toolkit-12-6
+
+Ensure libcusparseLt.so exists at /usr/local/cuda/lib64/:
+
+.. code-block:: sh
+
+    # if not exist, download and copy to the directory
+    wget https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-sbsa/libcusparse_lt-linux-sbsa-0.5.2.1-archive.tar.xz
+    tar xf libcusparse_lt-linux-sbsa-0.5.2.1-archive.tar.xz
+    sudo cp -a libcusparse_lt-linux-sbsa-0.5.2.1-archive/include/* /usr/local/cuda/include/
+    sudo cp -a libcusparse_lt-linux-sbsa-0.5.2.1-archive/lib/* /usr/local/cuda/lib64/
+
+
+Build torch_tensorrt
+~~~~~~~~~~~~~~
+
+
+Install bazel
+
+.. code-block:: sh
+
+    wget -v https://github.com/bazelbuild/bazelisk/releases/download/v1.20.0/bazelisk-linux-arm64
+    sudo mv bazelisk-linux-arm64 /usr/bin/bazel
+    chmod +x /usr/bin/bazel
+
+Install pip and required python packages
+
+.. code-block:: sh
+
+    # install pip
+    sudp apt install python3-pip
+
+.. code-block:: sh
+
+    # install setuptools with the version less than 71.*.* 
+    python -m pip install setuptools==70.2.0
```

Comment:
Won't pip do this for us?

Reply:
after flash the jetpack to 6.1 and installed the nvidia-jetpack developer kit, this is not installed, also if we directly install the latest setuptools, it will throw the following error: 
`TypeError: canonicalize_version() got an unexpected keyword argument 'strip_trailing_zero'`

Reply:
Can we make a jetson requirements.txt or something maybe?

Reply:
good idea, Let me add a jetson requirements.txt